src/Common/src/System/Net/Http/Http2/SR.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace System.Net.Http
+{
+    internal static partial class SR
```

Comment:
So this is how we want to do resource string sharing, right? Maybe leave a comment?